01-node-tutorial/answers/12-http.js

```diff
@@ -0,0 +1,8 @@
+const http = require('http');
+
+const server = http.createServer((req, res) => {
+  res.write('<h1 style="font-size: 80px">Welcome</h1>');
```

Comment:
The assignment specifies that you should have the `req` callback variable check for what url the user is visiting. What could you do to ensure there are multiple responses depending on what url you enter when visiting the server in the browser?

Reply:
Ok, got it. Haste makes waste. Here is the corrected code. Can you please check if this one is fine? If it is, do you want me to push it or I can push it with the next PR?

```
const http = require('http');

const server = http.createServer((req, res) => {
  if (req.url === '/') {
    res.write(`<h1 style="font-size: 80px">Welcome stranger</h1>
    <ul><li><a href="/about">about</a></li><li><a href="/not_about">not about</a></li><li><a href="yes_i_am">are you sure?</a></li></ul>`);
  } else if (req.url === '/about') {
    res.write(`<h1 style="font-size: 80px">This is a page. Just a page.<br>
    You can turn it <a href="/">back</a> any time</h1>`);
  } else if (req.url === '/lkejlkejfldsahgodsjdsafkjdsgfikjnflsdf') {
    res.write(`<h1 style="font-size: 80px">No way you ended up here just typing randomly!<br>
    You are definitely a Lucky Person!<br>
    Now it is time to go <a href="/">back</a>.</h1>`);
  } else {
    res.write(`<h1 style="font-size: 80px">Pardon, this may have been a wrong turn.<br>
    This <a href="/">link</a> will take you back safely</h1>`);
  };
  res.end();
});

server.listen(3000);
console.log("The server is listening on port 3000.");
```